protocol/messenger.go

```diff
@@ -2317,10 +2318,30 @@ func (m *Messenger) SendChatMessages(ctx context.Context, messages []*common.Mes
 
 // sendChatMessage takes a minimal message and sends it based on the corresponding chat
 func (m *Messenger) sendChatMessage(ctx context.Context, message *common.Message) (*MessengerResponse, error) {
+	var response MessengerResponse
+
 	displayName, err := m.settings.DisplayName()
 	if err != nil {
 		return nil, err
 	}
+	if err := utils.ValidateDisplayName(&displayName); err != nil {
+		// Somehow a wrong display name was saved. We need to update it before sending
+		pubKey, err := m.settings.GetPublicKey()
+		if err != nil {
+			return nil, err
+		}
+		replacementDisplayName := pubKey[:12]
```

Comment:
The `12` looks pretty random. I assume it's the max display name length, but maybe we have a constant defined somewhere?

Reply:
It is a bit random, but I asked Dario if he could send me a message since he also has an ENS and his display name was the 12 first characters of his pubkey. I have no idea where it came from, but it did make sense.

If there is a constant for that, I'm more than happy to use, but I don't know where it comes from.